File: Python/exercicio-10/solucao.py
def main():
    def tinta_necessaria():
        altura = float(input('Digite a altura da parede em metros: ' ))
        largura = float(input('Digite a largura da parede em metros: ' ))

        area = largura * altura
        quantidade_necessaria = area * 0.5

        print(f'A area da parede a ser pintada equivale a {area} m²')
        print(f'e a quantidade de tinta necessaria equivale a {quantidade_necessaria} litros.')
    

    tinta_necessaria()

main()
input("Pressione o Enter para sair ... ")

File: JavaScript/exercicio-10/solucao.js
const prompt = require("prompt-sync")();
const readline = require("readline-sync");

function main() {

  function calcularArea() {

    const altura = parseFloat(prompt('Digite a altura da parede em metros: ' ));
    const largura = parseFloat(prompt('Digite a largura da parede em metros: ' ));

    const area = largura * altura
    const tintaNecessaria =  area * 0.5

    console.log(`A area da parede a ser pintada equivale a ${area} m²`)
    console.log(`e a quantidade de tinta necessaria equivale a ${tintaNecessaria} litros.`)

  }

  calcularArea();

}

main();
readline.question('Pressione Enter para sair ... ')